runtimes/nodejs/nodejs.md

```diff
@@ -185,22 +185,34 @@ it from your environment: `process.env.MY_VARIABLE`.
 You can (and should) define the `NODE_ENV=production` environment variable as many frameworks
 rely on it for production mode.
 
-### Special env NPM_TOKEN
 
-Since April 2015, npmjs.com allows you to have private repositories. The bad news is,
```

Comment:
I wouldn't have deleted the whole bloc, keeping the explanation of why the customer may need those variables (private repositories)